uint8 mode
uint8 marker_id
uint8 layer
---
bool res
string response
uint8 error_code